R/utils.r

```diff
@@ -102,6 +102,19 @@ print_yaml <- function(x) {
   structure(x, class = "print_yaml")
 }
 
+pkgdown_field <- function(pkg, ...) {
+  field <- paste0(list(...), collapse = ".")
+  config <- src_path(
+    fs::path_rel(
+      pkgdown_config_path(pkg$src_path),
+      pkg$src_path
+    )
+  )
+  paste0(
+    crayon::bold(field), " in ", config, " (or in the `override` parameter)"
```

Comment:
I think mentioning override is likely to be confusing here, since it's an advanced feature.